src/ngAnimate/animateQueue.js

```diff
@@ -374,7 +379,7 @@ var $$AnimateQueueProvider = ['$animateProvider', function($animateProvider) {
             close();
             return runner;
           } else {
-            mergeAnimationOptions(element, existingAnimation.options, options);
+            mergeAnimationDetails(element, existingAnimation.options, options);
```

Comment:
How come we don't need to pass the animation object, but the options (considering all other invocation changed) ?